ksqldb-parser/src/main/antlr4/io/confluent/ksql/parser/SqlBase.g4

```diff
@@ -228,8 +228,8 @@ joinWindow
     ;
 
 withinExpression
-    : '(' joinWindowSize ',' joinWindowSize ')' # joinWindowWithBeforeAndAfter
-    | joinWindowSize # singleJoinWindow
+    : '(' joinWindowSize ',' joinWindowSize ')' (gracePeriodClause)?  # joinWindowWithBeforeAndAfter
```

Comment:
Are we planning to feature flag these changes in the event that not all the functionality makes it into the upcoming 0.20 release?

Reply:
ksqlDB already has a feature flag to disable the stream-stream join fix -- I think it makes sense to use it to fail if `GRACE` is used and the feature was disabled. Do you want to get this pulled into this PR or a later PR?

Reply:
I'll pull it into another PR